### PR TITLE
Fix test suites

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: tests/order_modifiers_integration.suite.dist.yml [hunk 1]
```diff
@@ -13,10 +14,8 @@ modules:
       tablePrefix: test_
       domain: %WP_DOMAIN%
       adminEmail: admin@%WP_DOMAIN%
-      title: 'Event Tickets Tests'
+      title: 'Event Tickets SLR Tests'
```
Comment (redscar) [inline tests/order_modifiers_integration.suite.dist.yml:17]: Should this be renamed?

File: tests/order_modifiers_integration/_bootstrap.php [hunk 2]
```diff
@@ -17,6 +18,8 @@
 $ticketable[] = 'post';
 tribe_update_option( 'ticket-enabled-post-types', array_values( array_unique( $ticketable ) ) );
 
+define( 'JSON_SNAPSHOT_OPTIONS', JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES );
```
Comment (redscar) [inline tests/order_modifiers_integration/_bootstrap.php:21]: What does this do?

File: tests/order_modifiers_integration.suite.dist.yml (hunk 1 above)
Comment (dpanta94) [reply tests/order_modifiers_integration.suite.dist.yml:17]: good catch!

File: tests/order_modifiers_integration/_bootstrap.php (hunk 2 above)
Comment (dpanta94) [reply tests/order_modifiers_integration/_bootstrap.php:21]: Controls the default output of JSON snapshot. It makes the beautiful and easy to read pretty much.

Whenever you use snapshot assertion you can use the method `assertMatchesJsonSnapshot`

File: tests/order_modifiers_integration/Orders/Stripe_Orders_Test.php [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+	use Order_Maker;
+	use Fee_Creator;
+
+	protected string $controller_class = Stripe_Fees::class;
+
+	/**
+	 * @after
+	 */
+	public function breakdown() {
+		tribe( $this->controller_class )->reset_fees_and_subtotal();
```
Comment (dpanta94) [inline tests/order_modifiers_integration/Orders/Stripe_Orders_Test.php:33]: Why not `$this->make_controller()->reset_fees_and_subtotal();` ?

File: tests/order_modifiers_integration/Orders/PayPal_Orders_Test.php [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+	use Order_Maker;
+	use Fee_Creator;
+
+	protected string $controller_class = Paypal_Fees::class;
+
+	/**
+	 * @after
+	 */
+	public function breakdown() {
+		tribe( $this->controller_class )->reset_fees_and_subtotal();
```
Comment (dpanta94) [inline tests/order_modifiers_integration/Orders/PayPal_Orders_Test.php:33]: Same comment as above here - otherwise all look good

**Pull request merged**